### PR TITLE
Add some buff and champ skill

### PR DESCRIPTION
**Pull request by requizm**
Add some buff and champ skill

### CODE REVIEW

**Comment by danil179** [comment]
Code need to be updated cause we changed classes to interfaces.

**Comment by requizm** [comment]
> Code need to be updated cause we changed classes to interfaces.

Okey, i will update as soon as possible

**Comment by requizm** [comment]
> Code need to be updated cause we changed classes to interfaces.

I dont see changes. Everything is still the same. How can I contact you?

**Comment by danil179** [comment]
> 
> 
> > Code need to be updated cause we changed classes to interfaces.
> 
> I dont see changes. Everything is still the same. How can I contact you?

Join the discord mentioned in the README if you want to contact LS developers.

File: Buffs/Snare/Snare.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+using LeagueSandbox.GameServer.GameObjects.Spells;
+using LeagueSandbox.GameServer.Scripting.CSharp;
+
+namespace Snare
+{
+    internal class Snare : IBuffGameScript
+    {
+        private UnitCrowdControl _crowd = new UnitCrowdControl(CrowdControlType.SNARE);
+
+        public void OnActivate(ObjAiBase unit, Spell ownerSpell)
```
Comment (danil179) [inline Buffs/Snare/Snare.cs:12]: All classes like `ObjAiBase` changed to interfaces like `IObjAiBase`.

**Review by danil179 — CHANGES_REQUESTED** [review]
See my comment about the classes->interfaces change. You can also compare your classes to other classes in this repo, and better, you can run GameServer and test if your file compile.

**Comment by requizm** [comment]
Deprecated issue

**Pull request closed without merging**